doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -3,39 +3,40 @@
 Version: 2.26  
 Doc. ID: UG-SYSPAR
```

Comment:
Add the following explanation of 'vanilla' term into 'Introduction' section:

Please note the term 'vanilla' in the documentation. In X-Road context vanilla means the X-Road without any of the country specific customizations, settings etc. The vanilla version of X-Road security server is installed with xroad-securityserver package. The country specific versions are installed with xroad-securityserver-XX where XX is the country code f.ex. FI or EE.